### PR TITLE
docs: improve README

### PR DESCRIPTION
**Pull request by luislhl**
### Description

Some small problems that were detected by users and discussed in Discord.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,20 +110,43 @@ poetry run make protos
...
 ```
 
+It may take a considerable amount of time for it to sync all the transactions in the network. To speed things up, read below.
+
+#### Speeding up the sync
+You can use database snapshots to speed things up.
+
+We provide both testnet and mainnet snapshots:
+- Testnet: https://hathor-public-files.s3.amazonaws.com/temp/testnet-data-20210707174940.tar.gz
+- Mainnet: https://hathor-public-files.s3.amazonaws.com/temp/data-20210624040233.tar.gz
```
Comment (luislhl) [inline README.md:120]: Check if it's possible to use a permanent link that just points to the most recent snapshot

**Review by pedroferreira1 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (luislhl) [reply README.md]: A proposed solution was commited

**Review by jansegre — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -110,20 +110,43 @@ poetry run make protos
 poetry run hathor-cli run_node --status 8080
 ```
 
+It may take a considerable amount of time for it to sync all the transactions in the network. To speed things up, read below.
+
+#### Speeding up the sync
+You can use database snapshots to speed things up.
+
+We provide both testnet and mainnet snapshots. You can get the link to the latest snapshots this way:
+- Testnet: `curl https://hathor-public-files.s3.amazonaws.com/temp/testnet-data-latest`
```
Comment (msbrogli) [inline README.md:119]: Should we add redirect urls? So we can redirect them accordingly when a new snapshot is available?

**Review by msbrogli — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (luislhl) [reply README.md:119]: The solution I proposed tries to solve this problem.

`https://hathor-public-files.s3.amazonaws.com/temp/testnet-data-latest` is just a one-line file containing the real snapshot url.

That's why one have to do `wget $(curl https://hathor-public-files.s3.amazonaws.com/temp/testnet-data-latest)` to get the snapshot

When a new snapshot is available, we just have to update the url in this file.

Comment (msbrogli) [reply README.md:119]: Great solution! Thanks!

**Pull request merged**